### ISSUE TITLE
0-conf possible attack using large transactions

### ISSUE THREAD

**Issue by CurtisMaves** [amiuhle/kasisto#33]
By making a very simple modification to the monero-wallet-rpc, one can generate a valid low priority transaction that is small enough to be relayed, but large enough to never be confirmed by the network. After looking through the code. I looked through the code and I didn't see a protection against this kind of attack.   

After 24 hours this transaction will drop from the mempool and the sender will be able to use the Monero again.

**Comment by amiuhle** [comment]
I agree, transactions like this shouldn't be propagated through the network.

**Comment by cmaves** [comment]
I made an issue on the monero repo.
https://github.com/monero-project/monero/issues/3189

**Comment by cmaves** [comment]
I'll leave this issue open until it is either fixed on the upstream or fixed in kasisto itself
